### PR TITLE
feat(twap): twap review order - pt4 - duration fields

### PR DESCRIPTION
**Pull request by alfetopito**
# Update

image

- Unhide Start time and hard code it to `now`

To be explained via tool tip

# Summary

Follow up to https://github.com/cowprotocol/cowswap/pull/2631
Implements part of https://github.com/cowprotocol/cowswap/issues/2577

Address PR comments from https://github.com/cowprotocol/cowswap/pull/2629

image

Calculating:
- Part time
- Total time

Hiding:
- Start time

# To Test

1. On a Gnosis Safe with the handler already updated, go to advanced
2. Fill in the form
3. Click on review
* You should see the fields above properly filled and calculated

# Notes

1. Styling definitely not ready

### CODE REVIEW

File: src/modules/trade/pure/ConfirmDetailsItem/index.tsx [hunk 1]
```diff
@@ -2,23 +2,22 @@ import { ReactNode } from 'react'
...
 import { RowFixed } from 'legacy/components/Row'
-import { MouseoverTooltipContent } from 'legacy/components/Tooltip'
 
 import { TextWrapper } from 'modules/swap/pure/Row/styled'
-import { StyledInfoIcon } from 'modules/swap/pure/styled'
 
 import { Content, Row, Wrapper } from './styled'
 
-type Props = {
+export type ConfirmDetailsItemProps = {
```
Comment (anxolin) [inline src/modules/trade/pure/ConfirmDetailsItem/index.tsx:12]: thanks, i probably didn't justify why I mentioned, but one reason is that is convenient to use when defining cosmos fixtures. Also, might be handy for some other things, and doesn't have a cost to add it (its just a type)

File: src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx [hunk 2]
```diff
@@ -19,29 +22,38 @@ export function TwapConfirmDetails(_props: TwapConfirmDetailsProps) {
...
       />
-      {/*TODO: implement logic for time fields*/}
-      <ConfirmDetailsItem tooltip="TODO: add tooltip" label={'Start time first' + partsSuffix} withArrow={false}>
-        {startTime}
-      </ConfirmDetailsItem>
+
+      {/* Start time */}
+      {/*<ConfirmDetailsItem tooltip="TODO: add tooltip" label={'Start time first' + partsSuffix} withArrow={false}>*/}
+      {/*  {startTime}*/}
+      {/*</ConfirmDetailsItem>*/}
```
Comment (anxolin) [inline src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx:46]: why removing it? we were going to make a proposal for the label, right?

File: src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 import { ConfirmDetailsItem } from 'modules/trade/pure/ConfirmDetailsItem'
 import { ReviewOrderModalAmountRow } from 'modules/trade/pure/ReviewOrderModalAmountRow'
 
 import { PartsState } from '../../state/partsStateAtom'
 
 export type TwapConfirmDetailsProps = {
   startTime: number | undefined
-  partDuration: number | undefined
+  partDuration: string
+  totalDuration: string
```
Comment (anxolin) [inline src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx:11]: are we sure the formatting is not part of the responsibility of this component? 
I would think, `partDuration: number`  + `numberOfParts: number` should be enough params

File: src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx (hunk 2 above)
Comment (alfetopito) [reply src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx:46]: So, keep it static `Now`?

I'll do. I just had understood we'd hide it until it's decided how to display it

**Review by shoom3301 — APPROVED** [review]
Cool!

File: src/modules/trade/pure/ConfirmDetailsItem/index.tsx [hunk 4]
```diff
@@ -30,11 +29,7 @@ export function ConfirmDetailsItem(props: Props) {
           <Row>
             <RowFixed>
               {label && <TextWrapper>{label}</TextWrapper>}
-              {tooltip && (
-                <MouseoverTooltipContent content={tooltip} wrap>
-                  <StyledInfoIcon size={16} />
-                </MouseoverTooltipContent>
-              )}
+              {tooltip && <InfoIcon content={tooltip} />}
```
Comment (nenadV91) [inline src/modules/trade/pure/ConfirmDetailsItem/index.tsx:32]: Awesome, I didn't remember this one.

**Review by nenadV91 — APPROVED** [review]

File: src/modules/trade/pure/ConfirmDetailsItem/index.tsx (hunk 4 above)
Comment (alfetopito) [reply src/modules/trade/pure/ConfirmDetailsItem/index.tsx:32]: Me neither, it was @shoom3301 's suggestion =P

**Pull request merged**